nip47/event_handler.go

```diff
@@ -47,7 +47,6 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 		return
 	}
 
-	ss, err := nip04.ComputeSharedSecret(event.PubKey, svc.keys.GetNostrSecretKey())
```

Comment:
Did you intend to remove this check? now on the right hand side we check the same error twice

Reply:
Maybe the second error check just needs to be removed